.github/workflows/incident-classification.yml

```diff
@@ -0,0 +1,107 @@
+name: Incident-Classification
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+    paths:
+      # Only run when incidents are modified
+      - "incidents/index.json"
```

Comment:
This is inconsistent. The incidents index is updated through another workflow, and no PR is opened for that. So we should either change the trigger or combine this with the existing workflow that updates the index.